src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -220,8 +253,12 @@ static ucs_status_t uct_cuda_copy_mem_alloc(uct_md_h md, size_t *length_p,
     }
 
     if (!uct_cuda_base_is_context_active()) {
-        ucs_error("attempt to allocate cuda memory without active context");
-        return UCS_ERR_NO_DEVICE;
+        ucs_debug("attempt to allocate cuda memory without active context");
```

Comment:
Maybe rephrase the message, as it now represents a "lazy init" rather than some "wrong condition".

Reply:
I would also print this message from uct_cuda_copy_md_set_ctx, when it succeeds

Reply:
Updated the message and moved it to uct_cuda_copy_md_set_ctx.